src/elm/Session/Guest.elm

```diff
@@ -126,7 +126,7 @@ view thisMsg page ({ shared } as model) content =
                   else
                     text ""
                 , div
-                    [ class "min-h-stretch flex flex-col"
+                    [ class "min-h-screen flex flex-col"
```

Comment:
oh I see @amiskov 

But i wonder why i couldn't reproduce this behavior from my machine, only after resizing the browser (before this commit)

Reply:
I tried to understand this but with no success... 🤷🏻‍♂️ Even `min-h-stretch` is not working anymore, but there were reasons for me to use it earlier! Maybe we changed something in the markup in previous commits in the past couple of months, but I didn't find anything suspicious...